daemon/indexing/migrations/20180815061230-add-listing-table.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+var fs = require('fs');
+var path = require('path');
+var Promise;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+  Promise = options.Promise;
+};
+
+exports.up = function(db) {
```

Comment:
This could be refactored to something like this: 
```
exports.up = function(db) {
  await executeSql(db, '20180815061230-add-listing-table-up.sql')
};

exports.down = function(db) {
  await executeSql(db, '20180815061230-add-listing-table-down.sql')
};

async executeSql(db, sqlFilePath) {
  var filePath = path.join(__dirname, 'sqls', sqlFilePath);
  return new Promise( function( resolve, reject ) {
    fs.readFile(filePath, {encoding: 'utf-8'}, function(err,data){
      if (err) return reject(err);
      console.log('received data: ' + data);

      resolve(data);
    });
  })
  .then(function(data) {
    return db.runSql(data);
  });
};
}
```

Reply:
Perhaps even moving executeSql function out of migration file into some sort of utils might make sense

Reply:
@sparrowDom This code is actually auto-generated by the db-migrate tool :)
See https://db-migrate.readthedocs.io/en/latest/Getting%20Started/installation/

Reply:
haha ok, didn't know that :)